docs/general/ddl/data-types.rst

```diff
@@ -1987,7 +1988,7 @@ Where ``string`` describes the interval using one of the recognized formats:
 | Standard SQL format  | ``3 4:05:06``         | 3 days 4 hours      |
 | (day-time)           |                       | 5 minutes 6 seconds |
 +----------------------+-----------------------+---------------------+
-| `PostgreSQL interval | ``1 year 2 months     | 1 year 2 months     |
+| `PostgreSQL interval | ``1 year 2 months``   | 1 year 2 months     |
```

Comment:
This should be reverted, the backticks are closing at the end of the expression (6 seconds).

Reply:
Done. Thanks.